static/css/responsive.css

```diff
@@ -0,0 +1,160 @@
+@media screen and (max-width: 1030px) {
+  .sidebar-left {
+    position: fixed;
+    background: #000000;
+    top: 0;
+    width: 100%;
+    height: 6em;
+    z-index: 2;
+  }
+
+  .sidebar-right {
+    display: none;
+  }
+
+  .header-title {
+    display: flex;
+    margin: 0;
+    padding: 0;
+  }
+
+  .header-title div {
+    flex: 1;
+  }
+
+  .header-title .category-list li {
+    display: inline-block;
+  }
+
+  .header-title .category-list li a,
+  .header-title .tags-list li a {
+    font-size: 24px;
+  }
+
+  .header-title .title {
+    display: none;
+  }
+
+  .header-title .avatar {
+    margin: 0;
+    width: 4em;
+    height: 4em;
+    border-width: 2px;
+  }
+
+  .header-title .tagline {
+    display: none;
+  }
+
+  .tags-list {
+    display: none;
+  }
+
+  .title-container, .tagline-container, .social-container {
+    display: none;
+  }
+
+  .content {
+    margin: 0;
+    margin-top: 7em;
+  }
+
+  .avatar {
+    width: 250px!important;
+  }
+  article {
+    margin: 0 2em;
+  }
+  .project_title {
+    font-size: 30px;
+    padding: 0 0 30px;
+
+  }
+  .recentPost_title {
+    font-size: 40px!important;
+  }
+  .col-xs-3 {
+    max-width: 0%!important;
+    flex-basis: 0%!important;
```

Comment:
remove `!important`, you should not overwrite `col-xs`, please use the library original built in way to do this.